server/player.lua

```diff
@@ -554,8 +567,8 @@ function CreatePlayer(playerData, Offline)
     ---@param val any
     function self.Functions.SetMetaData(meta, val)
         if not meta or type(meta) ~= 'string' then return end
-        if meta == 'hunger' or meta == 'thirst' then
-            val = val > 100 and 100 or val
+        if meta == 'hunger' or meta == 'thirst' or meta == 'stress' then
+            Player(self.PlayerData.source).state:set(meta, math.clamp(val, 0, 100), true)
```

Comment:
Pretty sure you gotta use `math = lib.math` with this right?

Reply:
Works fine for me 🤔 🤷🏻